cmd/api/main.go

```diff
@@ -19,13 +20,13 @@ func main() {
 	cfg := config.Load()
 	log := app.Logger(cfg.App.LogLevel)
 	book := addressbook.NewAddressBook(log, config.AddressPath, config.JettonPath, config.CollectionPath)
+	imgGenerator := image.NewImgGenerator()
 
 	storage, err := litestorage.NewLiteStorage(log, litestorage.WithPreloadAccounts(cfg.App.Accounts), litestorage.WithTFPools(book.TFPools()))
 	if err != nil {
 		log.Fatal("storage init", zap.Error(err))
 	}
-
-	h, err := api.NewHandler(log, api.WithStorage(storage), api.WithAddressBook(book))
+	h, err := api.NewHandler(log, api.WithStorage(storage), api.WithAddressBook(book), api.WithPreviewGenerator(imgGenerator))
```

Comment:
doesn't have any sense because inside you check is option null and set default generator

Reply:
✅